Cosign >=2.0.0 needs non-interactivity

When configuring `docker_signs` / `signs` with `cosign`, and `cosign` is >=2.0.0, `--yes` argument is needed.

An example log with failing _Release_ action:

```
Using payload from: dist/peribolos-owners-syncer_0.1.2_SHA256SUMS
Generating ephemeral keys...
Retrieving signed certificate...
Successfully verified SCT...

	Note that there may be personally identifiable information associated with this signed artifact.
	This may include the email address associated with the account with which you authenticate.
	This information will be used for signing this artifact and will be stored in public transparency logs and cannot be removed later.

By typing 'y', you attest that you grant (or have permission to grant) and agree to have this information stored permanently in transparency logs.
Are you sure you would like to continue? [y/N] Error: signing dist/peribolos-owners-syncer_0.1.2_SHA256SUMS: upload to tlog: user declined the prompt
main.go:74: error during command execution: signing dist/peribolos-owners-syncer_0.1.2_SHA256SUMS: upload to tlog: user declined the prompt
    • took: 1s
  ⨯ release failed after 3m19s               error=sign: cosign failed: exit status 1: Using payload from: dist/peribolos-owners-syncer_0.1.2_SHA256SUMS
```

GoReleaser config:

```yaml
...
# signs our docker image
# https://goreleaser.com/customization/docker_sign
docker_signs:
- cmd: cosign
  env:
  - COSIGN_EXPERIMENTAL=1
  artifacts: images
  output: true
  args:
  - 'sign'
  - '${artifact}'
...
```

GitHub action:
```
...
      - name: Publish release
        uses: goreleaser/goreleaser-action@f82d6c1c344bcacabba2c841718984797f664a6b #4.2.0
        with:
          version: latest
          args: release --clean
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
...
```

--- a/.goreleaser.yaml
+++ b/.goreleaser.yaml
@@ -89,3 +89,5 @@ docker_signs:
     args:
     - 'sign'
     - '${artifact}'
+    - "--yes" # needed on cosign 2.0.0+
+

--- a/.goreleaser.yaml
+++ b/.goreleaser.yaml
@@ -61,6 +61,7 @@ signs:
     - '--output-certificate=${certificate}'
     - '--output-signature=${signature}'
     - '${artifact}'
+    - "--yes" # needed on cosign 2.0.0+
   artifacts: checksum
   output: true
 

